lib/blather/stanza/x.rb

```diff
@@ -254,22 +254,25 @@ def label=(label)
       #
       # @param [String]
       def value
-        if self.namespace
-          content_from 'ns:value', :ns => self.namespace.href
+        children = if self.namespace
+          xpath('ns:value', ns: self.namespace.href)
         else
-          content_from :value
+          xpath(:value)
         end
+        return children.first&.content if children.length < 2
+        children.map(&:content)
```

Comment:
This appears to be special handling for a case where more than one child exists. Would you add a spec to show an example of how this broke?

Reply:
Added spec